recipe/emailpassword/emaildelivery/index.js

```diff
@@ -0,0 +1,6 @@
+"use strict";
```

Comment:
Update playground file to add how the user will use:
- importing
- overriding

For all recipes and delivery services